src/fu-util.c

```diff
@@ -2845,6 +2846,9 @@ main (int argc, char *argv[])
 	if (no_history)
 		priv->flags |= FWUPD_INSTALL_FLAG_NO_HISTORY;
 
+	/* start polkit tty agent to listen for password requests */
+	fu_polkit_agent_open ();
```

Comment:
does this need to run on every run of the app?

Reply:
My thought being for example it wouldn't run in non-interactive mode

Reply:
Yes, it gets killed after the command quits. It's exactly the same logic as pkcon has been using for ~ 5 years.